skbio/diversity/alpha/_base.py

```diff
@@ -847,6 +848,51 @@ def osd(counts):
     counts = _validate_counts_vector(counts)
     return observed_otus(counts), singles(counts), doubles(counts)
 
+@experimental(as_of="0.4.0")
```

Comment:
I used conda to install the dependancies listed in the travis-ci folder. I am now getting this new error after running `make test`.

```
...
Ran 1313 tests in 64.961s

OK
flake8 skbio setup.py checklist.py
./checklist.py
check-manifest
make: check-manifest: No such file or directory
make: *** [test] Error 1
```

I don't think the issue is with flake8 because this runs fine: `flake8 skbio setup.py checklist.py`